typespec/Applications.Dapr/bindings.tsp

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@typespec/openapi";
+import "@azure-tools/typespec-autorest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-providerhub";
+
+import "../radius/v1/ucprootscope.tsp";
+import "../radius/v1/resources.tsp";
+import "./common.tsp";
+import "../radius/v1/trackedresource.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Autorest;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+
+namespace Applications.Dapr;
+
+@doc("Dapr binding portable resource")
+model DaprBindingResource
```

Comment:
**Note:** Should I rename this from "DaprBindings" to "Binding" as per the design notes? Some Dapr components use the "Dapr" prefix, such as [statestore](https://github.com/radius-project/radius/blob/f118751e4d719971217b55d1292f495b273aa78f/typespec/Applications.Dapr/stateStores.tsp#L39), while others do not, such as [secretstore](https://github.com/radius-project/radius/blob/f118751e4d719971217b55d1292f495b273aa78f/typespec/Applications.Dapr/secretStores.tsp#L39).